.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,33 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="cryostatio/reviewers"
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          if [ -n "$reviewers" ]; then
+            gh pr edit ${{ github.event.issue.number }} --add-reviewer $reviewers
+          else
+            echo "Requesting review from @cryostatio/reviewers team"
+            gh pr edit ${{ github.event.issue.number }} --add-reviewer cryostatio/reviewer
```

Comment:
I think this should be `--add-reviewer cryostatio/reviewerS` plural: https://github.com/orgs/cryostatio/teams/reviewers